components/icons/images/tier1/video-assignment.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="18" height="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
+  <g fill="#494C4E" fill-rule="nonzero">
```

Comment:
Can you remove the fill rules? `<d2l-icon>` will add this colour back in, but if you specify it inside the SVG it can't be changed from the outside (e.g. on hover to celestine).

Reply:
Hmm... I see `<g fill="#494c4e"` in a whole bunch.

The difference is mine is using `fill-rule="nonzero"`
While the others have this `<g fill="#494c4e" fill-rule="evenodd">`

Is that what you're referring to?

Reply:
Ah sorry nevermind, forgot we automatically remove `#494c4e` at runtime -- you're good!